src/benchmark/operators/sort_benchmark.cpp

```diff
@@ -3,20 +3,113 @@
 #include "benchmark/benchmark.h"
 
 #include "../micro_benchmark_basic_fixture.hpp"
+#include "SQLParser.h"
+#include "SQLParserResult.h"
+#include "expression/expression_functional.hpp"
+#include "hyrise.hpp"
+#include "logical_query_plan/lqp_translator.hpp"
+#include "operators/limit.hpp"
 #include "operators/sort.hpp"
 #include "operators/table_wrapper.hpp"
+#include "sql/sql_pipeline_builder.hpp"
+#include "sql/sql_translator.hpp"
+#include "synthetic_table_generator.hpp"
+
+#include "micro_benchmark_utils.hpp"
 
 namespace opossum {
 
-BENCHMARK_F(MicroBenchmarkBasicFixture, BM_Sort)(benchmark::State& state) {
-  _clear_cache();
+static std::shared_ptr<Table> generate_custom_table(const size_t row_count, const ChunkOffset chunk_size,
+                                                    const DataType data_type = DataType::Int,
+                                                    const std::optional<float> null_ratio = std::nullopt) {
```

Comment:
Is it necessary to make this optional? Wouldn't `null_ratio = 0.0f` serve the same purpose?